docs/essentials/app-actions.md

```diff
@@ -1,33 +1,32 @@
 ---
-title: "Xamarin.Essentials: App Actions"
-description: "The Accelerometer class in Xamarin.Essentials lets you create and respond to app shortcuts from the app icon."
-author: jamesmontemagno
-ms.author: jamont
-ms.date: 01/04/2021
-no-loc: [Xamarin.Forms, Xamarin.Essentials]
+title: "Essentials: App Actions"
+description: "The AppActions class in Microsoft.Maui.Essentials lets you create and respond to app shortcuts from the app icon."
+ms.date: 08/02/2021
+no-loc: ["Microsoft.Maui", "Microsoft.Maui.Essentials", "AppDelegate.cs", "AppActions"]
 ---
 
-# Xamarin.Essentials: App Actions
+# App Actions
 
-The **AppActions** class lets you create and respond to app shortcuts from the app icon.
+The <xref:Microsoft.Maui.Essentials.AppActions> class lets you create and respond to app shortcuts from the app icon. App shortcuts are helpful to users because they allow you, as the app developer, to present them with additional ways of starting your app. For example, if you were developing an email and calendar app, you could present two different app actions, one to open the app directly to the current day of the calendar, and another to open to the email inbox folder.
 
 ## Get started
 
 [!INCLUDE [get-started](includes/get-started.md)]
 
-To access the **AppActions** functionality the following platform specific setup is required.
+To access the `AppActions` functionality the following platform specific setup is required.
 
 # [Android](#tab/android)
 
 Add the intent filter to your `MainActivity` class:
 
 ```csharp
-[IntentFilter(
-        new[] { Xamarin.Essentials.Platform.Intent.ActionAppAction },
-        Categories = new[] { Android.Content.Intent.CategoryDefault })]
-public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
+[IntentFilter(new[] { Microsoft.Maui.Essentials.Platform.Intent.ActionAppAction },
+              Categories = new[] { Android.Content.Intent.CategoryDefault })]
+[Activity(Theme = "@style/Maui.SplashTheme", MainLauncher = true, ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation | ConfigChanges.UiMode | ConfigChanges.ScreenLayout | ConfigChanges.SmallestScreenSize)]
```

Comment:
I'd remove this line. It will already have been added to the MainActivity class by the templates.